styleguide/source/assets/scss/01-atoms/_links.scss

```diff
@@ -67,6 +67,11 @@ a {
   }
 }
 
+//Adds underlines to all links inside of p tags
+p a {
+  text-decoration: underline;
```

Comment:
Are all links that don't have a specific class going to need this or only items that are inside `<p>` tags? I'm concerned that this could get buggy when dealing with WYSIWYG content or other content.

Reply:
only links that are inside of p tags are going have underlines.

Reply:
let's revisit this when we implement the WYSIWYG  editors. It could get funky